docs/content/plugins/_index.md

```diff
@@ -8,5 +8,9 @@ Porter. [Plugins are very different from mixins][vs], which give you building
 blocks for authoring bundles. There are a couple [types of plugins][types] and
 each plugin binary may contain multiple implementations.
 
+See the [Discoverability Guide][discoverability] on how to search for available plugins and/or
```

Comment:
This is great. How about we add a "Find Plugins" link under Plugins in the Nav too?

Reply:
Oops, good reminder. Should be added now.